src/collective/easyform/browser/view.py

```diff
@@ -346,6 +346,22 @@ def __call__(self, fname=None, fset=None):
         return super(EasyFormInlineValidationView, self).__call__(fname, fset)
 
 
+class GetSaveDataAdaptersView(BrowserView):
+
+    def __call__(self, *args, **kwargs):
+        """return all contained save data adapters"""
+        from collective.easyform.api import get_actions
+        from collective.easyform.interfaces import ISaveData
+        self.context = EasyFormForm(self.context, self.request)
```

Comment:
Do we need the form to be a class variable? I guess `view = EasyFormForm(self.context, self.request)` is less confusing in the long run.

Reply:
No, you're absolutely right... I was sloppy